pkg/trace/obfuscate/sql_tokenizer.go

```diff
@@ -490,7 +490,7 @@ exit:
 }
 
 func (tkn *SQLTokenizer) scanString(delim rune, kind TokenKind) (TokenKind, []byte) {
-	buffer := &bytes.Buffer{}
+	filtered := bytes.NewBuffer(tkn.buf[:0])
```

Comment:
```suggestion
	buf := bytes.NewBuffer(tkn.buf[:0])
```
Why not `buf`? It's standard, shorter, and it's what it was before.